ADR/0038-integration-service-composite-removal.md

```diff
@@ -0,0 +1,31 @@
+# 38. Integration service removes composite snapshots and logic around them
+
+* Date started: 2024-07-10
+
+## Status
+
+
+## Context
+
+Composite snapshots main goal was to prevent race condition when teams merged multiple PRs to multiple components
+of the same application at nearly the same time. This concept was confusing for users and we managed to simplify it
+by immediate promotion to GCL using override snapshots. Users also ran into the issue with GCL deadlock. This was 
+caused by multiple components present in GCL causing tests for a new component fail if they were bundled into a snapshot.
+This deadlock prevented the relase of new version of the application.
+
+
+## Decision
+
+Introduction of override snapshots should prevent this race condition and GCL deadlock with much simpler concept to understand,
+basically replacing the composite snapshots logic.
```

Comment:
Can you describe what an override snapshot is in any more detail?

Reply:
Added few sentences regarding override snapshot, let me know it that suffices.